text/0000-Project-Goals-2025h1.md

```diff
@@ -0,0 +1,662 @@
+| [Evaluate approaches for seamless interop between C++ and Rust](https://rust-lang.github.io/rust-project-goals/2025h1/seamless-rust-cpp.html)                       | [Tyler Mandry][]         | [compiler], [lang], [libs-api]                      |
+| [Experiment with ergonomic ref-counting](https://rust-lang.github.io/rust-project-goals/2025h1/ergonomic-rc.html)                                                   | [Santiago Pastorino][]      | [lang]                                              |
+| [Expose experimental LLVM features for GPU offloading](https://rust-lang.github.io/rust-project-goals/2025h1/GPU-Offload.html)                                      | [Manuel Drehwald][]          | [compiler], [lang]                                  |
+| [Extend pubgrub to match cargo's dependency resolution](https://rust-lang.github.io/rust-project-goals/2025h1/pubgrub-in-cargo.html)                                | [Jacob Finkelman][]          | [cargo]                                             |
+| [Externally Implementable Items](https://rust-lang.github.io/rust-project-goals/2025h1/eii.html)                                                                    | [Mara Bos][]         | [compiler], [lang]                                  |
+| [Field Projections](https://rust-lang.github.io/rust-project-goals/2025h1/field-projections.html)                                                                   | [Benno Lossin][]         | [compiler], [lang]                                  |
+| [Finish the libtest json output experiment](https://rust-lang.github.io/rust-project-goals/2025h1/libtest-json.html)                                                | [Ed Page][]           | [testing-devex]                                     |
+| [Implement Open API Namespace Support](https://rust-lang.github.io/rust-project-goals/2025h1/open-namespaces.html)                                                  | ![Help Wanted][] | [cargo], [compiler]                                 |
+| [Implement restrictions, prepare for stabilization](https://rust-lang.github.io/rust-project-goals/2025h1/restrictions.html)                                        | [Jacob Pratt][]         | [compiler], [lang]                                  |
+| [Improve state machine codegen](https://rust-lang.github.io/rust-project-goals/2025h1/improve-rustc-codegen.html)                                                   | [Folkert de Vries][]      | [compiler], [lang]                                  |
+| [Instrument the Rust standard library with safety contracts](https://rust-lang.github.io/rust-project-goals/2025h1/std-contracts.html)                              | [Celina G. Val][]        | [compiler], [libs]                                  |
+| [Integration of the FLS into the Rust Project](https://rust-lang.github.io/rust-project-goals/2025h1/spec-fls-integration.html)                                     | [Joel Marcey][]      | [release], [spec]                                   |
```

Comment:
```suggestion
| [Integration of the FLS into the Rust Project](https://rust-lang.github.io/rust-project-goals/2025h1/spec-fls-publish.html)                                     | [Joel Marcey][]      | [release], [spec]                                   |
```

Reply:
re:release ask here, it's not clear to me that release is the right team. @JoelMarcey, perhaps we can sync offline or otherwise discuss what you're looking for? Release generally doesn't own the "artifact pipeline", only the public-facing bits of it (i.e., kicking it off, attaching release notes, blog post). If we need another book published that's probably more of a T-infra/bootstrap ask, or perhaps just not an ask at all (if it works the same as every other book ~anyone can land a PR adding one).

Reply:
@Mark-Simulacrum Just [commented in Zulip](https://rust-lang.zulipchat.com/#narrow/channel/399173-t-spec/topic/Spec.20infra) as well. I had thought the `t-spec` team believed `t-release` may be necessary for tooling integration that could be needed for FLS publishing. If that's not true, I can have a conversation with `t-spec` at our meeting tomorrow.

Reply:
During our sync meeting today we determined that the only ask needed here was "standard reviews" from the bootstrap team (and we did not expect any particular load or challenge in doing those reviews). @JoelMarcey to update the goal. Resolving this thread.

Reply:
https://github.com/rust-lang/rust-project-goals/pull/241

Reply:
Do I need to make any updates to this RFC?